### PR TITLE
CaseDetailPane StackedBlock: Filter analysis_group by sample

### PR DESCRIPTION
**Pull request by alexkb0009**

### CODE REVIEW

**Review by sbreiff — CHANGES_REQUESTED** [review]
I tested this a little on local, and its a big improvement, but there seems to a be a little bug still.

I uploaded a family that has a WES Trio analysis, a WGS Trio, and a WGS Quad analysis. 
The proband case for the WGS-Trio looks good: https://gyazo.com/65efe5c3debe5f6e9980ce81aa88320f
The proband case for the WGS-Group is missing the father's case, which had appeared correctly in the Trio case: https://gyazo.com/9ef0c93f8db4e68063b9e65d9992cd96

**Review by sbreiff — APPROVED** [review]
I tested this branch out again and the problem I described earlier was gone this time. I'm ok to merge (when tests pass I guess?) as it seems like a random issue, but will report if I see anything like this on prod in the future.

**Review by Bianca-Morris — APPROVED** [review]

File: src/encoded/static/scss/encoded/modules/_tables.scss [hunk 1]
```diff
@@ -251,7 +251,8 @@ table.matrix {
 						So, to improve visual appeal, we can and should align the label to top edge.
 						This is done with an align-items override below.
 					*/
-					align-items: stretch; /* overrides default align-items: center */
+
+					/* align-items: stretch; */ /* overrides default align-items: center */
```
Comment (alexkb0009) [inline src/encoded/static/scss/encoded/modules/_tables.scss:255]: Will remove this more fully later or adjust for different (.name) element to have align-self: stretch after testing more.

Comment (Bianca-Morris) [inline src/encoded/static/scss/encoded/modules/_tables.scss:255]: Might make sense to get rid of these comments if we're not going to be using this, but it's not a super big deal. Especially if it's something that may be revisited/tweaked in future.

Comment (Bianca-Morris) [reply src/encoded/static/scss/encoded/modules/_tables.scss:255]: Ha. You beat me to it.

**Pull request merged**